crates/core/tedge_actors/src/servers/message_boxes.rs

```diff
@@ -339,4 +339,55 @@ mod tests {
             Ok(())
         );
     }
+
+    #[tokio::test]
```

Comment:
Good to have these two new tests actually detecting the issue. However, it would could to put them in a test module that doesn't require `cfg(feature = "test-helpers")`.

Reply:
I pushed a fixup commit to fix that https://github.com/thin-edge/thin-edge.io/pull/3291/commits/f0926246fb4e0f282c7fd7cb328967fa1c0c1258